docs/source/backends/javascript.rst

```diff
@@ -8,6 +8,7 @@ output an HTML file will also generate a basic HTML document with the
 generated code inside a ``<script>`` tag; the other distinction is on the ffi
 that will be explained below.
 
+**Note**: The JavaScript codegen uses the new BigInt, hence Node.js 10.4 or higher is required.
```

Comment:
Moved here since @dunhamsteve mentioned

> anything prior to node 10 hit end of life by 2019, so I'm not sure that bit is relevant anymore. BigInt is widely supported, but the BigInt requirement might be relevant to someone targeting older browsers or an embedded javascript with more limited support.

and "SlayerOfTheBad" mentioned

> Someone targeting older browsers will likely polyfill any of their JS before serving it anyways though